src/software/firmware/srcs/alarm_controller.cpp

```diff
@@ -192,5 +219,12 @@ void AlarmController::runAlarmEffects(uint16_t p_centiSec) {
         digitalWrite(PIN_LED_YELLOW, LED_YELLOW_INACTIVE);
     }
 
+    if (unsnooze) {
+        m_snoozeTime = 0u;
+        for (uint8_t i = 0; i < ALARMS_SIZE; i++) {
+            m_snoozedAlarms[i] = false;
+        }
+    }
```

Comment:
If I understand correctly, if there is a new alarm (same or different priority) we unsnooze all alarms? Even if the priority of new alarm is bellow the priority of snoozed alarms.

Reply:
Yes! That's what we decided with @cmdm yesterday morning

Reply:
Ok. Seems I forget this part of our meeting. Sorry.